### PR TITLE
Added test scripts for ReportsPage and did refactoring

### PR DESCRIPTION
**Pull request by StalloneSylvester**

### CODE REVIEW

File: src/test/java/com/paypal/base/BaseTest.java [hunk 1]
```diff
@@ -22,6 +23,7 @@ public class BaseTest extends EnvironmentSetter {
     protected String browser;
     protected String browserVersion;
     protected LoginPage loginPage;
+    protected String downloadsDirectoryPath = "C:/Users/Amardeep/Downloads";
```
Comment (mDaveMehr) [inline src/test/java/com/paypal/base/BaseTest.java]: avoid path dependency. Instead use system.getProperty("user.dir")

File: src/test/java/com/paypal/base/BaseTest.java [hunk 2]
```diff
@@ -52,19 +53,10 @@ public void setUp() {
...
-    public void pauseBrowser(int seconds) {
-        try {
-            Thread.sleep(seconds * 1000L);
-        } catch (InterruptedException e) {
-            throw new RuntimeException(e);
-        }
-    }
-
     public Transaction getTransaction() {
         CreditCard card = new CreditCard("John", "376680816376961",
```
Comment (mDaveMehr) [inline src/test/java/com/paypal/base/BaseTest.java]: Why username, credit card number are visible here

File: src/test/java/com/paypal/base/BaseTest.java (hunk 1 above)
Comment (StalloneSylvester) [reply src/test/java/com/paypal/base/BaseTest.java]: Merci Dave. I will use that.

File: src/test/java/com/paypal/base/BaseTest.java (hunk 2 above)
Comment (StalloneSylvester) [reply src/test/java/com/paypal/base/BaseTest.java]: Thanks for pointing this out. I will move them into properties file.

**Pull request merged**